src/constants.py

```diff
@@ -1,10 +1,21 @@
 from decimal import Decimal
+import os
 
 # Datasource
-URL_API_THE_GRAPH = 'https://api.thegraph.com/subgraphs/name/gnosis/protocol'
-URL_UI_THE_GRAPH = 'https://thegraph.com/explorer/subgraph/gnosis/protocol'
+if 'NETWORK' in os.environ and os.environ['NETWORK'] == "xdai":
```

Comment:
Maybe u can factor out `'NETWORK' in os.environ and os.environ['NETWORK']`, so there's no so much repetition


Also, maybe it's good to define a dictionary with all possible values of `NETWORK`. Then you get the value from the constant dictionary (you raise an error if there's no value)

This way, also, u handle better the case where someone uses a unsupported network, that in this case u would default to mainnet (but I think is better raising an error)